src/game/behaviors/sparkle_spawn_star.inc.c

```diff
@@ -66,7 +70,7 @@ void bhv_spawned_star_loop(void) {
             o->activeFlags |= ACTIVE_FLAG_INITIATED_TIME_STOP;
             o->oAngleVelYaw = 0x800;
             if (o->oBehParams2ndByte == SPAWN_STAR_POS_CUTSCENE_BP_SPAWN_AT_MARIO) {
-                set_home_to_mario();
+                spawned_star_set_target_above_mario();             
```

Comment:
```suggestion
                spawned_star_set_target_above_mario();
```
trailing whitespace moment